Avalonia.Templates.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>0.10.0-rc1</PackageVersion>
+    <PackageId>Avalonia.Templates</PackageId>
+    <Title>Avalonia Templates</Title>
+    <Description>Templates for creating Avalonia applications and libraries.</Description>
+    <PackageTags>dotnet-new;templates;avalonia</PackageTags>
+    <TargetFramework>net5.0</TargetFramework>
+    <IncludeContentInPack>true</IncludeContentInPack>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <ContentTargetFolders>content</ContentTargetFolders>
+  </PropertyGroup>
```

Comment:
`<NoPackageAnalysis>true</NoPackageAnalysis>` should reduce build noise

`<NoDefaultExcludes>true</NoDefaultExcludes>` will allow you to add files like .gitignore

Maybe `<TargetFramework>netstandard2.0</TargetFramework>` is better? this will allow collecting not only from .net5 sdk

Also, we can consider the following improvements:
```xml
<PackageProjectUrl>https://github.com/AvaloniaUI/avalonia-dotnet-templates</PackageProjectUrl>
<RepositoryUrl>https://github.com/AvaloniaUI/avalonia-dotnet-templates.git</RepositoryUrl>

<OutputPath>bin</OutputPath>
```

Reply:
By the way, consider simplifying .gitgnore to this state

```
.idea/
.vscode/
.vs/

bin/
obj/

*.user
```

This is quite enough for modern dotnet.

Reply:
Thanks @rstm-sf - good suggestions - I've applied them!